src/niclips/figures/multi_view.py

```diff
@@ -160,7 +160,10 @@ def slice_video(
     cmap: str = "gray",
     fontsize: int = 14,
 ) -> None:
-    """Save video scrolling through range of slices."""
+    """Save video scrolling through range of slices.
+
+    NOTE: For 4D image, currently have to set idx to None
```

Comment:
Hm, I'm confused. I thought the behavior was if `img` is 4d, `idx` is used to slice along the time dimension. If `idx is None`, then the middle volume is used. Can you explain more what you mean here?

Reply:
Ahh okay. I see where this is going wrong then and it is due to the fixture. The mask that it generates is almost always going to be 1 voxel since most of data will always be smaller than the mean.  

For testing purposes, I can generate a more random matrix which should get around this. I think it might also help to catch and raise a more meaningful error message (eg no data in mask or something). I also think that we might encounter this more as we extend to diffusion data.

Reply:
Okay, updated the test fixture!